lib/rspec-console/exceptions.rb

```diff
@@ -0,0 +1,8 @@
+module RSpecConsole
+  class VersionError < StandardError
+    'Please use RSpec 2.9.10 or later'
```

Comment:
yeah, the spec was weak on this -- i'll expand it to cover the message explicitly.